packages/react-router/src/index.tsx

```diff
@@ -183,6 +183,7 @@ export {
   PathParamError,
   getInitialRouterState,
   defaultSerializeError,
+  getRouteMatch,
```

Comment:
We avoid exporting this to the public, yes?

Reply:
I removed the export